package.json

```diff
@@ -6,13 +6,13 @@
   "type": "module",
   "scripts": {
     "mvnbuild": "npm run build",
-    "mvntest": "npm test",
+    "mvntest": "eslint --format checkstyle --output-file target/eslint-warnings.xml; vitest run --reporter junit",
     "build": "vite build --mode production",
-    "build:dev": "vite build --mode development --watch",
+    "build:dev": "vite build --mode production --watch",
```

Comment:
I think it's better to use production even during development to ensure a more consistent experience.

The build is very fast here anyway.

https://vite.dev/guide/rolldown.html#why-introducing-a-full-bundle-mode